OnBoardingKit/OBPrivacyLinkButton.h

```diff
@@ -0,0 +1,9 @@
+#import <UIKit/UIKit.h>
+
+API_AVAILABLE(ios(13.0))
+@interface OBPrivacyLinkButton : UIButton
+
+- (instancetype)initWithCaption:(NSString *)caption buttonText:(NSString *)buttonText image:(UIImage *)image imageSize:(CGSize)imageSize useLargeIcon:(BOOL)useLargeIcon API_DEPRECATED("Use the method with displayLanguage parameter instead", ios(13.0, 15.6));
+- (instancetype)initWithCaption:(NSString *)caption buttonText:(NSString *)buttonText image:(UIImage *)image imageSize:(CGSize)imageSize useLargeIcon:(BOOL)useLargeIcon displayLanguage:(NSString *)displayLanguage API_AVAILABLE(ios(16.0));
```

Comment:
I think it may be better to hold off on these until iOS 16 is out of beta. There's a chance iOS 15.7 may come out, and a chance this iOS 16 declaration may be updated before release.

Reply:
Fair point. 😉